app/views/registrations/show.html.erb

```diff
@@ -62,9 +62,9 @@
     <div class="alert alert-danger"><%= @registration_form.errors.full_messages.to_sentence %></div>
   <% end %>
 
-  <div data-controller="registration-tabs">
+  <% is_form = f.object.instance_of?(RegistrationForm) %>
```

Comment:
Move this up a few lines so it's available in the `<div>` that mounts the registration-tabs controller.